modules/javafx.base/src/test/java/test/javafx/beans/property/ListPropertyBaseTest.java

```diff
@@ -814,6 +817,56 @@ public void testToString() {
         assertEquals("ListProperty [name: My name, value: " + value1 + "]", v4.toString());
     }
 
+    @Test
+    public void testBindingLeak() {
+        JMemoryBuddy.memoryTest(checker -> {
+            // given
+            System.out.println("Start collection: " + FXCollections.observableArrayList());
```

Comment:
By the way, can these `println` statements in various tests be removed?

Reply:
Done! I've removed the 3 print statements!